docker_build_dependency_image.sh

```diff
@@ -46,6 +46,11 @@ if [[ -z ${MODE} ]]; then
   echo "Default MODE=${MODE}"
 fi
 
+if [[ -z ${DEVICE} ]]; then
+  export DEVICE=tpu
+  echo "Default DEVICE=${DEVICE}"
```

Comment:
Can we move this echo outside of the if statement so that it always prints the default device type?

Reply:
Wouldn't it be confusing if we set `DEVICE=gpu`, but we are echoing `Default DEVICE=tpu`? Or do you suggest we always echo the device information we are currently using?

Reply:
We won't get the device as TPU if we set to GPU. This piece of code only sets the device type as TPU is no device type is assigned. So if you assign device type as GPU, you will get GPU.

Reply:
I see what you are suggesting. I add additional line to always print current DEVICE.